.gitignore

```diff
@@ -12,3 +12,6 @@
 *.out
 
 bin/*
```

Comment:
We don't create this directory anymore (we can remove this).

Reply:
Done.